cubedash/templates/dataset.html

```diff
@@ -219,6 +229,10 @@ <h3 class="followed">
             });
             L.control.zoom({position: "bottomright"}).addTo(map);
             map.fitBounds(dataset_data.getBounds(), {animate: false, maxZoom: 5});
+            const has_thumbnail = '{{ thumbnail_url|length >= 1 }}';
```

Comment:
How about instead of the work around with a `thumbnail_url` list. We implement a new jinja2 filter for extracting a thumbnail URL from a dataset.

It will be ever so slightly less efficient, but IMO cleaner and less likely to get broken in the future if/when these templates get refactored.

See https://github.com/opendatacube/datacube-explorer/blob/develop/cubedash/_filters.py#L120-L122 for the implementation of the `dataset_file_paths` filter used above.

I'm ambivalent about keeping the implementation in `_utils.py` and the filter in `_filters.py`, I don't see why some filters can contain the own implementation... but on the other hand, it would keep the code more consistent to continue with that pattern. 🖐

Reply:
Agreed; done